.github/workflows/push.yml

```diff
@@ -34,7 +34,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node: [12.0.0, 12.x, 14.x, 16.x, 18.x]
+        node: [14.0.0, 14.x, 16.x, 18.x]
```

Comment:
How is `14.0.0` different from `14.x`?

Reply:
14.0.0 is 14.0.0, 14.x is latest 14 line.